### DISCUSSION TITLE
I want to upload multiple images in one input, what do I do?

### DISCUSSION THREAD

**Discussion by BaraaDev** [spatie/laravel-medialibrary#2727] [Q&A] [answered]
I want to upload multiple images in one input, what do I do?
However, I want it to be one model

**Comment by erikn69** [comment, marked as answer]
There is many plugins js for that, only you have to handle the upload on server

**Reply by BaraaDev** [reply]
But I mean the backend, not the frontend

**Reply by erikn69** [reply]
>one input

That sounds like html input In front 🤣, sorry

**Reply by BaraaDev** [reply]
take a look here
`if ($request->hasFile('image')) {
            $users
                ->addMediaFromRequest('image')
                ->UsingName($users->name)
                ->UsingFileName("$users->name.jpg")
                ->toMediaCollection('images');
        }`
